tests/integration/org_test.go

```diff
@@ -204,9 +204,7 @@ func TestTeamSearch(t *testing.T) {
 	var results TeamSearchResults
 
 	session := loginUser(t, user.Name)
-	csrf := GetCSRF(t, session, "/"+org.Name)
 	req := NewRequestf(t, "GET", "/org/%s/teams/-/search?q=%s", org.Name, "_team")
-	req.Header.Add("X-Csrf-Token", csrf)
```

Comment:
To help review:
The `"/"+org.Name` page doesn't contain CSRF token.
However, it's not required to provide the token for GET requests.

Reply:
GET request doesn't need CSRF token.